makefiles/boot/riotboot.mk

```diff
@@ -74,9 +74,11 @@ $(BINDIR_APP)-slot1.hdr: OFFSET=$(SLOT1_IMAGE_OFFSET)
 # Generic target to create a binary files for both slots
 riotboot: $(SLOT_RIOT_BINS)
 
+# Same as CLEAN for the submake riotboot/bootlader/% targets
+RIOTBOOT_CLEAN = $(if $(filter clean, $(MAKECMDGOALS)),riotboot/bootloader/clean,)
 # riotboot bootloader compile target
 riotboot/flash-bootloader: riotboot/bootloader/flash
-riotboot/bootloader/%: $(BUILDDEPS)
+riotboot/bootloader/%: $(BUILDDEPS) | $$(filter-out $$@,$$(RIOTBOOT_CLEAN))
```

Comment:
or just add `| $(CLEAN)` here?

Reply:
ah, no, it must be `riotboot/bootloader/clean`

Reply:
maybe  `riotboot/bootloader/clean` can be added to "clean" as prerequisite?

Reply:
That does the trick as well.